hieradata/site/dev.yaml

```diff
@@ -103,3 +103,5 @@ accounts::user_list:
 # Stop iptables by default - the default rules are highly restrictive to the
 # point of harm and we don't have a meaningful permission set to make this useful.
 firewall::ensure: "stopped"
+
+letsencrypt::server: "https://acme-staging.api.letsencrypt.org/directory"  # testing url
```

Comment:
I know is a better practice, but you plan to merge the "staging" server? Also,  I thought the server was https://acme-v02.api.letsencrypt.org/directory

Reply:
Yes, this only applies to the dev site.